internal/test/wmcb_test.go

```diff
@@ -218,6 +229,41 @@ func TestWMCBUnit(t *testing.T) {
 	assert.NotContains(t, string(out), "FAIL")
 }
 
+// getKubeClient gets the kubernetes client needed for testing.
+func getKubeClient() (*k8sclient.Clientset, error) {
+	kubeconfig := os.Getenv("KUBECONFIG")
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	kclient, err := k8sclient.NewForConfig(config)
+	if err != nil {
+		return nil, err
+	}
+	return kclient, nil
+}
+
+// TestWMCBCluster runs the cluster tests for the nodes
+func TestWMCBCluster(t *testing.T) {
+	//TODO: Transfer the WMCB binary to the Windows node and approve CSR for the Windows node.
+	// I want this to be moved to another test. We've another card for this, so let's come back
+	// to that later(WINC-82). As of now, this test is limited to check if the taint has been
+	// applied to the Windows node and skipped for now.
+	client, err := getKubeClient()
+	require.NoError(t, err, "error getting kubeclient")
```

Comment:
Please print the error that getKubeClient() is returning for future debugging purposes :)
```
require.NoErrorf(t, err, "error getting kubeclient: %s", err)
```

Reply:
Done.